tests/test_meta_commands.py

```diff
@@ -548,13 +548,16 @@ def test_integer_out_of_range(hlwm):
                 .expect_stderr('out of range')
 
 
-def test_raise_winid_missing(hlwm):
+def test_raise_lower_winid_missing(hlwm):
     hlwm.call_xfail('raise') \
         .expect_stderr('raise: not enough arguments\n')
 
+    hlwm.call_xfail('lower') \
+        .expect_stderr('lower: not enough arguments\n')
+
 
 def test_raise_invalid_winid(hlwm):
```

Comment:
Shouldn't we test both cases in here? You replaced raise with lower, so raise is not tested with an invalid winid

Reply:
Oh, right, thanks! I've fixed it.